lib/chef/knife.rb

```diff
@@ -530,10 +530,10 @@ def humanize_http_exception(e)
         client_api_version = version_header["request_version"]
         min_server_version = version_header["min_version"]
         max_server_version = version_header["max_version"]
-        ui.error "The version of #{Chef::Dist::PRODUCT} that Knife is using is not supported by the #{Chef::Dist::PRODUCT} server you sent this request to"
+        ui.error "The version of #{Chef::Dist::PRODUCT} that Knife is using is not supported by the server you sent this request to"
```

Comment:
Any chance while you're in here @Tensibai you can add periods at the end of all these sentences.